src/kw_remote.sh

```diff
@@ -210,6 +215,18 @@ function rename_remote()
   fi
 }
 
+function list_remotes()
```

Comment:
Please, add documentation on top of this function. Check `deploy.sh` for a reference.